tools/src/_utils/index.ts

```diff
@@ -53,7 +53,7 @@ export function determine_possible_schema_types (doc: OpenAPIV3.Document, schema
   if (schema?.anyOf !== undefined) return collect_all(schema.anyOf)
   if (schema?.oneOf !== undefined) return collect_all(schema.oneOf)
 
-  if (schema == null || Object.keys(schema).filter(k => k !== 'description').length == 0) return SCHEMA_OBJECT_TYPES
+  if (schema == null || Object.keys(schema).filter(k => k !== 'description' && k !== 'title').length == 0) return SCHEMA_OBJECT_TYPES
```

Comment:
Could you please explain this change?

Reply:
If nothing is specified in the schema other than "informational" properties (ie. description or title) than it can be any type.